database/upload.go

```diff
@@ -7,15 +7,24 @@ import (
 	"gitlab.com/NebulousLabs/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
-	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo"
 )
 
 // Upload ...
 type Upload struct {
-	ID        primitive.ObjectID  `bson:"_id,omitempty" json:"_id"`
-	UserID    primitive.ObjectID  `bson:"user_id,omitempty" json:"user_id"`
-	SkylinkID primitive.ObjectID  `bson:"skylink_id,omitempty" json:"skylink_id"`
-	Timestamp primitive.Timestamp `bson:"timestamp" json:"timestamp"`
+	ID        primitive.ObjectID `bson:"_id,omitempty" json:"_id"`
+	UserID    primitive.ObjectID `bson:"user_id,omitempty" json:"user_id"`
+	SkylinkID primitive.ObjectID `bson:"skylink_id,omitempty" json:"skylink_id"`
+	Timestamp time.Time          `bson:"timestamp" json:"timestamp"`
+}
+
+// UploadResponseDTO is the DTO we send as response to the caller.
+type UploadResponseDTO struct {
+	ID        string    `bson:"string_id" json:"id"`
```

Comment:
same remark with the `bson`

Reply:
same answer. I don't think we should keep duplicating these comments when the cases are identical.